### ISSUE TITLE
custom text alignment middle by default

### ISSUE THREAD

**Issue by aurkaxi** [ibonn/python_avatars#2]
hey, great library. felt robust enough with those installations things.

ok so in custom text the text is not aligned in the middle by default. it's slightly on the right. 
![image](https://user-images.githubusercontent.com/72933395/213366208-0ad5b988-d488-428d-8e1c-f3aac8439abd.png)

**Comment by ibonn** [comment]
hello @ashroyxi 
I will try to fix this as soon as possible, it shouldn't be too complex

**Comment by ibonn** [comment]
Solved, will be officially added to version **1.4.0**. If you want to use it now install the library from [this branch](https://github.com/ibonn/python_avatars/tree/2-center-shirt-text)
```
pip install git+https://github.com/ibonn/python_avatars.git@2-center-shirt-text
```

#### Test cases
![avatar_1](https://user-images.githubusercontent.com/37736154/213936870-2dee57db-ff93-4941-a1c9-5d2b166f84b4.png)
> 1 character string

![avatar_3](https://user-images.githubusercontent.com/37736154/213936876-402f21cd-f35b-45fa-9cc3-7f2b2cc9363b.png)
> 3 character string

![avatar_7](https://user-images.githubusercontent.com/37736154/213936881-a95971da-2d78-45ad-a87a-b542a89e8058.png)
> 7 character string

![avatar_11](https://user-images.githubusercontent.com/37736154/213936887-c0561ce5-0a41-4cd7-ae0e-07d981daf7a4.png)
> 11 character string

**Comment by aurkaxi** [comment]
Thanks King
